src/components/__tests__/Login.test.js

```diff
@@ -0,0 +1,5 @@
+import {render,screen,cleanup} from '@testing-library/react';
+import Login from "../Login";
+test('should render Login Panel', () => {
+  const {debug} =   render(<Login/>)
```

Comment:
no use of debug can be removed,
in login page inputs can be tested with fire event methods (check my git for reference)
and for UI testing snapshot cases can be also added here